src/activate_linux.c

```diff
@@ -181,6 +201,8 @@ int main(int argc, char *argv[]) {
                 HELP("-s scale\tScale ratio (float)");
                 HELP("-H offset\tMove overlay horizontally (integer)");
                 HELP("-V offset\tMove overlay  vertically  (integer)");
+                HELP("-S screen_list\tSelect screens to display the message");
```

Comment:
```suggestion
                HELP("-S screens\tSelect screens to display the message");
```
The text is too long to fit in one default width `\t`
![image](https://user-images.githubusercontent.com/24810600/173237552-7db99616-9c93-4c90-8841-984e88e1d6d5.png)